lib/situation_room/check/outage.ex

```diff
@@ -0,0 +1,34 @@
+defmodule SituationRoom.Check.Outage do
+  @moduledoc """
+  Module that checks if there is an outage
+  """
+  alias SituationRoom.Outages
+  @behaviour SituationRoom.Check.Behaviour
+
+  @impl true
+  def run(%SituationRoom.Site.Check{} = check) do
```

Comment:
Can we use a more descriptive function name here?

Reply:
My idea on all these checks was to have them all have a @behaviour and thus have the same method name on all of them so we can do something like: 

```elixir
head_checks = [
  Checks.StatusCode,
  Checks.Outage
]

head_checks
|> Enum.map(fn module -> apply(module, "run", site))
```

but we could still probably rename them to something else